lib/screens/updater.rb

```diff
@@ -0,0 +1,81 @@
+class UpdaterScreen
```

Comment:
In Ruby the convention is that the file name should match the class name.
So this class should either be named `Updater` or the file should be renamed to `updater_screen.rb`.